tools/modules/software/module_bazarr.sh

```diff
@@ -1,10 +1,13 @@
 module_options+=(
-	["module_bazarr,author"]="@igorpecovnik"
+	["module_bazarr,author"]=""
+	["module_bazarr,maintainer"]="@igorpecovnik"
 	["module_bazarr,feature"]="module_bazarr"
+	["module_bazarr,example"]="install remove purge status help"
 	["module_bazarr,desc"]="Install bazarr container"
-	["module_bazarr,example"]="install remove status help"
-	["module_bazarr,port"]="6767"
 	["module_bazarr,status"]="Active"
+	["module_bazarr,doc_link"]="https://wiki.bazarr.media/"
+	["module_bazarr,group"]="Downloaders"
+	["module_bazarr,port"]="6767"
 	["module_bazarr,arch"]="x86-64,arm64"
```

Comment:
IMO for consistency 

["module_bazarr,ports"]="6767" some modules have multiple 



comma or space separated throughout. 

`["module_bazarr,example"]="install,remove,status,help"` 
or 
`["module_bazarr,arch"]="x86-64 arm64"` 

for cvs would need to change the IFS 
`IFS=' ' read -r -a commands <<< "${module_options["module_bazarr,example"]}"`
to
`IFS=',' read -r -a commands <<< "${module_options["module_bazarr,example"]}"`

Reply:
then lets keep spaces as this is now.